src/screens/Moderation/index.tsx

```diff
@@ -201,6 +202,24 @@ export function ModerationScreenInner({
 
   return (
     <View style={[a.pt_2xl, a.px_lg, gtMobile && a.px_2xl]}>
+      {isDeclaredUnderage && (
+        <View style={[a.pb_2xl]}>
+          <Admonition type="tip" style={[a.pb_md]}>
+            <Trans>
+              Your declared age is under 18. Some settings below may be
+              disabled. If this was a mistake, you may edit your bithdate in
+              your{' '}
+              <InlineLinkText
+                to="/settings/account"
+                label={_(msg`Account Settings`)}>
```

Comment:
```suggestion
                label={_(msg`Go to account settings`)}>
```
nit: Suggest adjusting label to match one used in similar copy, reducing string duplication:

https://github.com/bluesky-social/social-app/blob/122a46891a7f912c8e2777bae00c4b1f64154257/src/components/ageAssurance/AgeAssuranceAdmonition.tsx#L86-L97